src/app/containers/PageHandlers/withData/shouldRender/index.js

```diff
@@ -0,0 +1,39 @@
+import pathOr from 'ramda/src/pathOr';
+import {
+  getPassportHome,
+  isValidPassportHome,
+} from '../../../../lib/utilities/passport';
+
+// checks for data, status, setting default status if not found
+const constructRenderObject = data => ({
+  status: pathOr(null, ['status'], data) || 500,
+  pageData: pathOr(null, ['pageData'], data),
+});
+
+// checks for pageData, 200 status and if home service from article data fits the service locale
+const shouldRender = (data, service, passportHomesOverride = []) => {
```

Comment:
I dont think `shouldRender` needs to be moved in this PR. If we just pass `passportHomes` to `isValidPassportHome` all the testing can be contained to the `isValidPassportHome` function.

Reply:
I'm having trouble writing tests for this change, and I'm not sure I'll be able to do it in time for the presentation.

- shallowRender does not work for
```
<ServiceContext.Provider value={portugueseService}>
            <WithDataHOC {...validPortugueseData} />
          </ServiceContext.Provider>
```

and deep render is throwing a Garbage collection error.